scripts/build-bundle.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+node_modules/.bin/webpack --config webpack.config.bundle.js &
+node_modules/.bin/webpack --config webpack.config.bundle.js --minify &
+wait
```

Comment:
I only learned about it today. The `&` at the end of the two previous lines runs them as background jobs, i.e. in parallel. The `wait` command will end the script when both have finished.